react/README.md

```diff
@@ -48,3 +48,10 @@ For example, to add XYZ to the `package.json` file in the `react` folder while d
 (This advice has come from https://github.com/BretFisher/node-docker-good-defaults)
 
 ## Testing
+Testing is done with React Testing Library and Jest.
+
+Test files are located in the same directory as the files they are testing. For example, CreateBox.test.js and CreateBox.tsx are both located in react\src\views\CreateBox. 
```

Comment:
Can you wrap filenames and file paths in backticks such that the Markdown is rendered as 'code font'?. Also in the last paragraph.

In the filepath I'd propose to use forward slashes. All other filepaths in our READMEs I could find have forward slashes.